extension/src/popup/views/DisplayBackupPhrase.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from "react";
+import CopyToClipboard from "react-copy-to-clipboard";
+import styled from "styled-components";
+
+import { POPUP_WIDTH } from "constants/dimensions";
+import { emitMetric } from "helpers/metrics";
+import { useMnemonicPhrase } from "popup/helpers/useMnemonicPhrase";
+
+import { COLOR_PALETTE } from "popup/constants/styles";
+import { METRIC_NAMES } from "popup/constants/metricsNames";
+import { download } from "popup/helpers/download";
+
+import { BasicButton, BackButton } from "popup/basics/Buttons";
+
+import { Toast } from "popup/components/Toast";
+import { ActionButton } from "popup/components/mnemonicPhrase/ActionButton";
+
+import { HeaderContainerEl, HeaderEl } from "popup/views/UnlockBackupPhrase";
+
+import Download from "popup/assets/download.png";
+import Copy from "popup/assets/copy.png";
+
+const MnemonicDisplayEl = styled.div`
+  background: ${COLOR_PALETTE.primaryGradient};
+  border-radius: 30px;
+  color: ${({ isBlurred }: { isBlurred: boolean }) =>
+    isBlurred ? "transparent" : "#fff"};
+  font-size: 1.125rem;
+  margin: 2rem 0 1rem;
+  padding: 1.68rem 2.31rem;
+  position: relative;
+  text-align: center;
+  text-shadow: ${({ isBlurred }: { isBlurred: boolean }) =>
+    isBlurred ? "0 0 5px rgba(0, 0, 0, 0.5)" : "none"};
+`;
+
+const DisplayTooltipEl = styled(BasicButton)`
+  color: ${COLOR_PALETTE.white};
+  font-size: 1rem;
+  position: absolute;
+  text-shadow: none;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const DisplayButtonsEl = styled.div`
+  margin-bottom: 2.5rem;
+  margin-right: 1rem;
+  position: relative;
+  text-align: right;
+
+  img {
+    margin-left: 0.5rem;
+  }
+`;
+const CopiedToastWrapperEl = styled.div`
+  right: 15.625rem;
+  position: absolute;
+`;
+const UnlockAccountEl = styled.div`
+  width: 100%;
+  max-width: ${POPUP_WIDTH}px;
+  box-sizing: border-box;
+  padding: 2rem 2.5rem;
+`;
+
+export const DisplayBackupPhrase = () => {
+  const mnemonicPhrase = useMnemonicPhrase();
+  const [isCopied, setIsCopied] = useState(false);
+  const [isBlurred, setIsBlurred] = useState(true);
+
+  return (
+    <UnlockAccountEl data-testid="display-mnemonic-phrase">
+      <HeaderContainerEl>
+        <HeaderEl>
+          <BackButton onClick={() => window.location.replace("/")} />
```

Comment:
Using `window.location.replace` to go back to main `account` page. I wasn't sure if it needed to go back to password prompt view.

Reply:
I like this going back to Account straightaway.

There is an interesting issue here, though. This works in localhost, but does not work when you build and run the extension. That's become the internal url structure for the extension is a little different.

In the extension, the internal url for `/` is actually `chrome-extension://lnkacgnnjdohikfldmbgdkmbancciicl/index.html#/`

If you replace this with `history.push({ pathname: "/" })` it will work in both localhost and the extension itself.
In fact, I think we should probably always use `history.push` instead of `window.location` just to be safe

Reply:
I just opened a PR that creates a helper method that uses `history.push` to nav. Why don't I merge that PR and then you can just use the helper method here 

https://github.com/stellar/lyra/pull/92/files

Reply:
Sounds good, @piyalbasu. Thank you!

Reply:
Updated, @piyalbasu